sdk/keyvault/Azure.ResourceManager.KeyVault/src/autorest.md

```diff
@@ -7,7 +7,7 @@ azure-arm: true
 library-name: KeyVault
 namespace: Azure.ResourceManager.KeyVault
 tag: package-2021-10
-output-folder: Generated/
+output-folder: $(this-folder)/Generated
```

Comment:
If I run the codegen command under `sdk/keyvault/Azure.ResourceManager.KeyVault`, what the real output folder?

Reply:
It will change nothing. I add this one because `Generated/` and `$(this-folder)/Generated` will have different behavior when we encounter the [code](https://github.com/Azure/autorest.csharp/blob/feature/v3/src/AutoRest.CSharp/Common/AutoRest/Plugins/GeneratedCodeWorkspace.cs#L112) here, which have influence in the customization. But yes, for this RP and this change, it has nothing to do.

Reply:
Then consider to have this added into the template

Reply:
I already did this, thank you!